ehsm_kms_service/ehsm_access_server.env

```diff
@@ -0,0 +1,5 @@
+#ehsm_access_server
+export EHSM_CONFIG_REPORTERS=""
+export EHSM_CONFIG_DENY_SIZE_LIMIT=2
```

Comment:
does 2 make sense?
add some #comment to each env. e.g.
EHSM_CONFIG_REPORTERS:  The whitelisted IP who can report violations to access the server.

Reply:
This file is just for local test run. I will delete it.